flows/update_neo4j.py

```diff
@@ -126,7 +109,9 @@ def create_or_update_concept_nodes(
         """
         UNWIND $batch AS concept
         MERGE (c:ConceptNode {wikibase_id: concept.wikibase_id})
-        SET c.preferred_label = concept.preferred_label
+        SET c.preferred_label = concept.preferred_label,
+            c.alternative_labels = concept.alternative_labels,
+            c.negative_labels = concept.negative_labels
```

Comment:
question: do we care about description/definition here? or, are you imagining a client side app using the graph would also have to pull from the concept store to fetch these?